components/appNavbar.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <nav class="navbar">
+    <ul class="navbar__list">
+      <li class="navbar__item">
+        <NuxtLink class="navbar__item-link" to="/">
```

Comment:
предлагаю еще одно правило в наш стайлгайд:
писать стили не через дефис а как camelCase
```suggestion
        <NuxtLink class="navbar__itemLink" to="/">
```
с дефисами можно запутаться в БЭМ

Reply:
)